src/helpers/webStorage.js

```diff
@@ -0,0 +1,26 @@
+//Common class for interacting with web storage
+export class WebStorage{
```

Comment:
You can just name the service as `SessionStorageService` and name the methods as just `setItem`, `getItem`, etc.